Cargo.toml

```diff
@@ -12,4 +12,9 @@ clap = "2.23.2"
 uuid = { version = "0.4", features = ["v4"] }
 dbus = "0.5.2"
 regex = "0.2"
-nix = "0.8.1"
\ No newline at end of file
+nix = "0.8.1"
+darksky = { version = "0.5.0", optional = true }
```

Comment:
I think it's fine to pull this dependency without conditional compilation. If weather isn't used it ends up only as a few extra MB on the output binary, not impacting performance. If there is another reason for having this as an optional dependency, tell me and you should document in the README how to disable the weather feature when compiling.